lib/constants.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  IGNORE_FILE: '.hyperdriveignore'
```

Comment:
(optionally) adding this in empty directories to prevent them from being removed from the drive.